nomos-services/chain/broadcast-service/src/lib.rs

```diff
@@ -80,6 +104,16 @@ where
                         error!("Could not subscribe to new blocks channel: {err:?}");
                     }
                 }
+                BlockBroadcastMsg::SubscribeBlendSession { result_sender } => {
+                    if let Err(err) = result_sender.send(self.blend_session.subscribe()) {
```

Comment:
Are we supposed to get one value immediately after subscribing to this endpoint? Or is that handled elsewhere?

Reply:
The retransmission is not implemented, I think we could hold the latest value and send it on subscription, would that be enough in your case?